cleanlab_studio/internal/api/api.py

```diff
@@ -574,6 +575,70 @@ def get_deployed_model_info(api_key: str, model_id: str) -> Dict[str, str]:
     return cast(Dict[str, str], res.json())
 
 
+def create_enrichment_project(
+    api_key: str,
+    dataset_id: str,
+    name: str,
+    target_column_in_dataset: str,
+) -> JSONDict:
+    """Create a new enrichment project."""
+    check_uuid_well_formed(dataset_id, "dataset ID")
+    request_json = dict(
+        dataset_id=dataset_id,
+        name=name,
+        target_column_in_dataset=target_column_in_dataset,
+    )
+    res = requests.post(
+        f"{enrichment_base_url}/projects",
+        headers=_construct_headers(api_key),
+        json=request_json,
+    )
+    handle_api_error(res)
+    return cast(JSONDict, res.json())
+
+
+def delete_enrichment_project(api_key: str, project_id: str) -> None:
+    """Delete an enrichment project."""
+    check_uuid_well_formed(project_id, "Enrichment Project ID")
+    res = requests.delete(
+        f"{enrichment_base_url}/projects/{project_id}", headers=_construct_headers(api_key)
+    )
+    handle_api_error(res)
+    return
+
+
+def get_enrichment_project(api_key: str, project_id: str) -> JSONDict:
+    """Get an existing enrichment project."""
+    check_uuid_well_formed(project_id, "Enrichment Project ID")
+    res = requests.get(
+        f"{enrichment_base_url}/projects/{project_id}",
+        headers=_construct_headers(api_key),
+    )
+    handle_api_error(res)
+    return cast(JSONDict, res.json())
+
+
+def get_enrichment_projects(api_key: str) -> List[JSONDict]:
```

Comment:
as low-level api, it doesn't need to fetch all projects. but provide a way to do that. 
i.e. `def list_enrichment_project(api_key: str, next_page: int = 1)`

Reply:
How about changing this to `list_all_enrichment_projects()` and adding a pagination API if we need it later?

Reply:
OK. 
This is not the best practice, just fyi

Reply:
>  This is not the best practice, just fyi

What do you think we should return to the user?